### PR TITLE
UI changes for various tickets

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look, these changes include changes for the following tickets:
- MM-2017 - Making the back button a bit bigger for Account Settings so that it’s easily tappable.
- MM-2045 - Styling the search spin icon for mobile.
- MM-2050 - Adjusting the modal-close button so that it’s positioned absolutely and doesn’t mess up the modal header.
- MM-2071 - Adjusting spacing on Slack import screens also improving the formatting a bit.
- MM-2101 - Adding a max-height to the LHS drop down menu so that it produces a scroll if there’s a lot of content.

### CODE REVIEW

**Comment by jwilander** [comment]
+1

**Comment by hmhealey** [comment]
Looks good to me

File: web/react/components/team_import_tab.jsx [hunk 1]
```diff
@@ -99,18 +97,18 @@ export default class TeamImportTab extends React.Component {
                         data-dismiss='modal'
                         aria-label='Close'
                     >
-                        <span aria-hidden='true'>&times;</span>
+                        <span aria-hidden='true'>{'&times;'}</span>
```
Comment (crspeller) [inline web/react/components/team_import_tab.jsx]: Will the &times; work inside the jsx string? It doesn't seem to work for me.

**Comment by asaadmahmoodspin** [comment]
@crspeller Yeah looks like it's not working with &times; good catch, anything we can add to that or just leave write it out without a jsx container?

**Comment by crspeller** [comment]
@asaadmahmoodspin A quick google reveals this page: https://facebook.github.io/react/docs/jsx-gotchas.html  Which explains you can ether insert the character directly, or use the UTF escape sequence. Searching for times reveals this page: http://www.fileformat.info/info/unicode/char/00d7/index.htm which tells us the escape sequence is `\u00D7` or gives us the charactor × to copy and paste in.

**Comment by asaadmahmoodspin** [comment]
Done @crspeller

**Comment by hmhealey** [comment]
+1

**Comment by crspeller** [comment]
Thanks @asaadmahmoodspin !

**Pull request merged**